### PR TITLE
Release version 1.0beta2

### PR DESCRIPTION
**Pull request by mark-a-potts**
## Describe your changes  
This PR has minor changes from the EPIC1.0 base. 
A new do_landDA_release.sh script is added with a more portable configuration and remove module loading/sourcing
New yaml files are added that work with the fv3-bundle version being installed in containers and on RDHPCS systems.

### CODE REVIEW

File: do_landDA.sh [hunk 1]
```diff
@@ -27,15 +28,23 @@ echo "reading DA settings from $config_file"
...
 
 source $config_file
 
 LOGDIR=${OUTDIR}/DA/logs/
 OBSDIR=${OBSDIR:-"/scratch2/NCEPDEV/land/data/DA/"}
 
-# executable directories
-INCR_EXECDIR=${LANDDADIR}/add_jedi_incr/exec/   
+# executables
+calcfIMS_EXEC=${BUILDDIR}/bin/calcfIMS.exe
```
Comment (ClaraDraper-NOAA) [inline do_landDA.sh]: same issue as previously, with finding the executables.

File: do_landDA.sh [hunk 2]
```diff
@@ -58,9 +67,11 @@ if [[ ! -e ${OUTDIR}/DA ]]; then
     mkdir ${OUTDIR}/DA/logs
     mkdir ${OUTDIR}/DA/hofx
 fi 
+export JEDIWORKDIR=${WORKDIR}/mem000/jedi
+OROG_PATH=${LANDDA_INPUTS}/forcing/${atmos_forc}/orog_files
```
Comment (ClaraDraper-NOAA) [inline do_landDA.sh:62]: Can we put this in the settings?

File: do_landDA.sh [hunk 3]
```diff
@@ -286,9 +297,8 @@ if [[ ${DAtype} == 'letkfoi_snow' ]]; then
     echo 'do_landDA: calling create ensemble' 
 
     # using ioda mods to get a python version with netCDF4
-    source ${LANDDADIR}/ioda_mods_hera
```
Comment (ClaraDraper-NOAA) [inline do_landDA.sh:289]: Can we retain the option to call this, if not EPIC?

File: do_landDA.sh [hunk 4]
```diff
@@ -307,17 +317,16 @@ if [[ ! -e Data ]]; then
 fi
 
 echo 'do_landDA: calling fv3-jedi' 
-source ${JEDI_EXECDIR}/../../../fv3_mods_hera
```
Comment (ClaraDraper-NOAA) [inline do_landDA.sh:310]: As above.

File: do_landDA.sh [hunk 5]
```diff
@@ -203,6 +213,7 @@ if [[ $do_DA == "YES" ]]; then
    sed -i -e "s/XXMP/${MP}/g" letkf_land.yaml
    sed -i -e "s/XXDP/${DP}/g" letkf_land.yaml
    sed -i -e "s/XXHP/${HP}/g" letkf_land.yaml
+   sed -i -e "s#DATAPATH#${OROG_PATH}#g" letkf_land.yaml
```
Comment (ClaraDraper-NOAA) [inline do_landDA.sh:207]: Should probably also add this to the hofx.

File: do_landDA.sh [hunk 6]
```diff
@@ -344,10 +353,9 @@ cat << EOF > apply_incr_nml
 EOF
 
     echo 'do_landDA: calling apply snow increment'
-    source ${LANDDADIR}/land_mods_hera
```
Comment (ClaraDraper-NOAA) [inline do_landDA.sh:347]: and again.

**Review by ClaraDraper-NOAA — CHANGES_REQUESTED** [review]
Thanks Mark. Just a couple of minor changes.

File: do_landDA.sh (hunk 2 above)
Comment (mark-a-potts) [reply do_landDA.sh:62]: Yes. I will do that.

File: do_landDA.sh (hunk 3 above)
Comment (mark-a-potts) [reply do_landDA.sh:289]: I really hesitate to leave these machine specific modules in a release. They support a legacy build which we are not documenting and not testing. I could add a variable called "USE_LEGACY" or something and put the module sourcing in if statements, but it is not very clean.

Comment (ClaraDraper-NOAA) [reply do_landDA.sh:289]: Let's just get rid of them then. I can do the Cmake v legacy check in develop.

**Review by ClaraDraper-NOAA — APPROVED** [review]

**Pull request merged**